lib/view.php

```diff
@@ -268,15 +268,21 @@ protected function get_config($name = null)
      * @param object $message
      * @return string 
      */
+
     protected function message_outter_styles($message)
     {
-        $styles = 'width:' . (int) $message->width . 'px;height:' . (int) $message->height . 'px;'
```

Comment:
The message popup is not centred in the page. See the screenshot
https://drive.google.com/file/d/0B_uR_iaMki44QTFCTHB3UFBveTQ/view?usp=sharing